hydra-gradle-core/src/main/java/com/pandora/hydra/LazyTestExcluder.java

```diff
@@ -43,13 +47,35 @@ public boolean isSatisfiedBy(FileTreeElement fileTreeElement) {
         }
     }
 
+    private void logTestBlackListIfSpecified() {
+        final HydraPluginExtension pluginExtension = project.getExtensions().getByType(HydraPluginExtension.class);
+        if (!pluginExtension.isLogTestExclusions()) {
+            return;
+        }
+
+        String hostname = System.getenv("NODE_NAME");
```

Comment:
Could you change this to be based off the slaveName in HydraPluginExtension instead?

Reply:
I think I tried that at first. The problem is slaveName is often explicitly unspecified on the HydraPluginExtension, so it comes back null. Or at least, because _I'm_ not specifying it there, it's coming back null. If that's unexpected, I can try it again.

Reply:
Oh ok, perhaps then just provide a fallback in case NODE_NAME isn't defined (assuming NODE_NAME is jenkins specific)